### PR TITLE
Mount /etc/pki to the OpenStack CCM container and minor test fixes

### PR DESCRIPTION
**Pull request by xmudrii**
**What this PR does / why we need it**:

CA certificates in the `/etc/ssl/certs` directory on CentOS 7 are only symlinks pointing to `/etc/pki`. We only mount `/etc/ssl/certs` in the OpenStack CCM container, so symlinks are not working at all. This is causing the OpenStack CCM to end up in a CrashLoopBack. To fix this issue, we are now mounting `/etc/pki` to the OpenStack CCM container.

Addionally, this PR includes some fixes from #2273 needed to make the new E2E tests work on OpenStack.

**What type of PR is this?**

/kind bug

**Does this PR introduce a user-facing change? Then add your Release Note here**:
```release-note
- Mount `/etc/pki` to the OpenStack CCM container to fix CrashLoopBackoff on clusters running CentOS 7 
- Example Terraform configs for OpenStack are no longer attaching a Floating IP address to the initial MachineDeployment. This matches the behavior of not attaching Floating IP addresses to the control plane nodes.
```

**Documentation**:
```documentation
NONE
```

### CODE REVIEW

File: examples/terraform/openstack/output.tf [hunk 1]
```diff
@@ -86,7 +86,7 @@ output "kubeone_workers" {
           securityGroups = [openstack_networking_secgroup_v2.securitygroup.name]
           network        = openstack_networking_network_v2.network.name
           subnet         = openstack_networking_subnet_v2.subnet.name
-          floatingIpPool = var.external_network_name
+          # floatingIpPool = var.external_network_name
```
Comment (ahmedwaleedmalik) [inline examples/terraform/openstack/output.tf:89]: We should probably revert this change?!

**Review by ahmedwaleedmalik — APPROVED** [review]
/approve

File: examples/terraform/openstack/output.tf (hunk 1 above)
Comment (xmudrii) [reply examples/terraform/openstack/output.tf:89]: We decided to keep this change. The release note is updated accordingly.

**Comment by xmudrii** [comment]
/override pull-kubeone-e2e-openstack-default-install-containerd-external-v1.24.3
We'll test this as part of #2273
